pkg/serving/service.go

```diff
@@ -38,6 +41,23 @@ func RevisionTemplateOfService(service *servingv1alpha1.Service) (*servingv1alph
 	return config.DeprecatedRevisionTemplate, nil
 }
 
+var charChoices = []string{
+	"b", "c", "d", "f", "g", "h", "j", "k", "l", "m", "n", "p", "q", "r", "s", "t", "v", "w", "x",
```

Comment:
I gotta ask... did you purposely remove some chars to reduce the chances of inappropriate words?  :-)

Reply:
@duglin see initial PR comment:

> Otherwise this is substantially similar to the server-side generateName(), up to and including our lack of vowels.